package.json

```diff
@@ -6,11 +6,11 @@
     "global": "^4.4.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^2.1.3",
     "yarn": "^1.22.19"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
@olegkolyaka78 did you have some issues starting your project?  I had to remove this parameter in order to run it.